### DISCUSSION TITLE
Recaptcha solving

### DISCUSSION THREAD

**Discussion by bl4ckh4nd** [Second-Hand-Friends/kleinanzeigen-bot#145] [Ideas]
Hi,

thank you for coding this useful tool!
One question: I've been using the chrome extension "Buster" in the past to solve captchas.

Could it be implemented for solving the captcha here as well? According to a medium article, it's actually pretty straightforward to implement it with Selenium.

**Comment by CaptainCat999** [comment]
Hello,

I have an issue with captcha. The bot keeps logging in even when it's already logged in, thus triggering the captcha. The whole point for me is to be away while the bot republishes the ad, but with the current code it doesn't seem possible...
